components/node/flags/flags.go

```diff
@@ -110,27 +110,27 @@ var (
 		Required: false,
 		Value:    0,
 	}
-	ProposerEnabledFlag = &cli.BoolFlag{
-		Name:    "proposer.enabled",
-		Usage:   "Enable proposing of new L2 blocks. A separate batch submitter has to be deployed to publish the data for syncers.",
-		EnvVars: prefixEnvVar("PROPOSER_ENABLED"),
-	}
-	ProposerStoppedFlag = &cli.BoolFlag{
-		Name:    "proposer.stopped",
-		Usage:   "Initialize the proposer in a stopped state. The proposer can be started using the admin_startProposer RPC",
-		EnvVars: prefixEnvVar("PROPOSER_STOPPED"),
-	}
-	ProposerMaxSafeLagFlag = &cli.Uint64Flag{
-		Name:     "proposer.max-safe-lag",
+	SequencerEnabledFlag = &cli.BoolFlag{
+		Name:    "sequencer.enabled",
+		Usage:   "Enable sequencing of new L2 blocks. A separate batch submitter has to be deployed to publish the data for verifiers.",
```

Comment:
Maybe after we decide to change `syncer` to `verifier`, it should be changed then.
```suggestion
		Usage:   "Enable sequencing of new L2 blocks. A separate batch submitter has to be deployed to publish the data for syncers.",
```